Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1153,7 +1209,9 @@ open class NavigationMapView: UIView {
         let shape = delegate?.navigationMapView(self, shapeFor: waypoints, legIndex: legIndex) ?? FeatureCollection(features: features)
         
         if route.legs.count > 1 {
+            removeAlternativeRoutes()
             routes = [route]
+            updateRouteDurations(along: routes)
```

Comment:
@Udumft, I wonder why you've added this line here? Is it required to update route duration annotation layers in case when showing waypoints only? The reason I'm asking is that we have a regression based on newly added snapshot tests in scope of https://github.com/mapbox/mapbox-navigation-ios/pull/3933. For example:

![Screen Shot 2022-06-13 at 3 45 09 PM](https://user-images.githubusercontent.com/1496498/173459822-7e4a4137-7cea-4426-b9b3-ef3533b0325f.png)

Reply:
TBH, I don't remember exactly. It was either to make previewing routes correctly display duration banners, or it was a part of my attempt to add duration banners for alternative routes, which I forgot to cleanup.